core/src/main/scala/org/ensime/config/EnsimeConfigProtocol.scala

```diff
@@ -28,17 +27,18 @@ object EnsimeConfigProtocol {
   private implicit val moduleFormat: SexpFormat[EnsimeModule] = cachedImplicit
   private implicit val configFormat: SexpFormat[EnsimeConfig] = cachedImplicit
 
-  def parse(config: String): EnsimeConfig = {
+  def parse(config: String, serverConfig: String): EnsimeConfig = {
```

Comment:
@hardmettle if you need the server configuration for parsing `.ensime` , construct a `EnsimeServerConfig` either by parsing it or from system properties.

Adding a `sexp` parser for `EnsimeServerConfig` should be as simple as adding an implicit `SexpFormat` for it.